### PR TITLE
Add proposal process

### PR DESCRIPTION
**Pull request by xla**
* Outline rationale and process in README.md
* Bootstrap proposal template

***
[Rendered template](https://github.com/radicle-dev/radicle-decisions/blob/xla/proposal-process/proposals/0000-template.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,39 @@
 
 ## What is this?
 
-`radicle-decisions` is the home for our proposed decisions that have been scrutinised by our team of developers.
-_Accepted decisions_ are expected to be ahered to and practised throughout the `radicle-dev` organisation.
-_Rejected decisions_ will be be left as a trace to what we have proposed and why it was ultimately rejected.
+`radicle-decisions` is the home for our proposed decisions that have been
+scrutinised by our team of developers.
+_Accepted decisions_ are expected to be ahered to and practised throughout the
```
Comment (kim) [inline README.md]:
```suggestion
_Accepted decisions_ are expected to be adhered to and practised throughout the
```

File: proposals/0000-template.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+* Title: *Desriptive title*
+* Champion: *@github handle/email@*
+* Converstaion (optional): *thread on radicle.community*
```
Comment (kim) [inline proposals/0000-template.md]:
```suggestion
* Conversation (optional): *thread on radicle.community*
```

File: proposals/0000-template.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+* Title: *Desriptive title*
```
Comment (kim) [inline proposals/0000-template.md]:
```suggestion
* Title: *Descriptive title*
```

**Review by kim — APPROVED** [review]
I like this. Very lightweight.

**Review by abbey-titcomb — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -4,6 +4,39 @@
...
+
+### Common decision to capture
+
+* programming techniques
+* architecture changes/patterns
+* tools and dependency recommendations
+* ecosystem
+
+### Process
+* Open a PR with a new proposal copied from the [template]()
```
Comment (NunoAlexandre) [inline README.md]: missing link here

File: README.md [hunk 5]
```diff
@@ -4,6 +4,39 @@
...
+
+## Proposals
+
+### Requirements
+
+The proposal process is meant to drive cross-cutting engineering improvements
+across Radicle. We want the bar for a proposal to be as low as possible - to
+encourage positive change and alignment among engineers. Therefore we see any
+proposal as valid as long as an engineer inside of Radicle sees the need to
+formalise or decide on a certain aspect off development. It can, but doesn't
```
Comment (NunoAlexandre) [inline README.md]:
```suggestion
formalise or decide on a certain aspect of development. It can, but doesn't
```

?

**Review by NunoAlexandre — APPROVED** [review]
Looks good!

**Review by geigerzaehler — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -4,6 +4,39 @@
...
+need to be, originate from a discussion on
```
Comment (rudolfs) [inline README.md]:
```suggestion
need to originate from a discussion on
```

File: README.md [hunk 7]
```diff
@@ -4,6 +4,39 @@
...
+### Common decision to capture
+
+* programming techniques
+* architecture changes/patterns
+* tools and dependency recommendations
+* ecosystem
+
+### Process
+* Open a PR with a new proposal copied from the [template]()
+* Assign yourself - as the assignee is in charge to see the proposal through
```
Comment (rudolfs) [inline README.md:36]:
```suggestion
* Assign yourself - as the assignee in charge to see the proposal through
```

**Review by rudolfs — APPROVED** [review]

**Review by geigerzaehler — APPROVED** [review]

**Review by CodeSandwich — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -4,6 +4,39 @@
...
+
+* programming techniques
+* architecture changes/patterns
+* tools and dependency recommendations
+* ecosystem
+
+### Process
+* Open a PR with a new proposal copied from the [template](https://github.com/radicle-dev/radicle-decisions/blob/master/proposals/0000-template.md)
+* Assign yourself - as the assignee is in charge to see the proposal through
+* Request `radicle-dev/engineering` as reviewers
```
Comment (FintanH) [inline README.md:37]: Maybe we can just add a CODEOWNERS file and add the team there

File: README.md [hunk 9]
```diff
@@ -4,6 +4,39 @@
...
+* Announce new proposal in the [development
+  section](https://radicle.community/c/development-discussion/9)
+* Keep the conversation in the PR to not lose information
+* Incorporate feedback
+* Get quorum for approval
```
Comment (FintanH) [inline README.md:42]: Should we decide here what designates a quorum?

**Review by FintanH — APPROVED** [review]
Looks good! Just a few additions that might make the process easier to manage :)

File: README.md [hunk 10]
```diff
@@ -4,6 +4,39 @@
 
 ## What is this?
 
-`radicle-decisions` is the home for our proposed decisions that have been scrutinised by our team of developers.
-_Accepted decisions_ are expected to be ahered to and practised throughout the `radicle-dev` organisation.
-_Rejected decisions_ will be be left as a trace to what we have proposed and why it was ultimately rejected.
+`radicle-decisions` is the home for our proposed decisions that have been
+scrutinised by our team of developers.
+_Accepted decisions_ are expected to be adhered to and practised throughout the
```
Comment (cloudhead) [inline README.md]:
```suggestion
_Accepted decisions_ are expected to be adhered to and practiced throughout the
```

**Review by cloudhead — APPROVED** [review]

File: proposals/0000-template.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+* implementation timeline/guideline (optional)
+
+# Drawbacks
+*Why should this not be done?*
+
+# Open Questions
+*Enumerate unknowns and unresolved questions.*
+
+# Future possibilities
+*Potential follow-ups and related work at a later point in time.*
```
Comment (FintanH) [inline proposals/0000-template.md:26]: I would actually add one more header `# Status` which can be **Rejected**, **Accepted**, or **Pending**

Comment (xla) [reply proposals/0000-template.md:26]: Alternatively: open PR == Pending, merged == accpeted, PR closed == rejected.

Comment (FintanH) [reply proposals/0000-template.md:26]: Well I think they should all be merged to preserve this statement: https://github.com/radicle-dev/radicle-decisions/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R11-R12

Comment (xla) [reply proposals/0000-template.md:26]: You sire, played well. This battle is yours but the war is not over just yet.

***
Will amend.

Comment (FintanH) [reply proposals/0000-template.md:26]: Heh :smile: :bowing_man:

**Review by massimiliano-mantione — APPROVED** [review]
LGTM!

**Comment by FintanH** [comment]
I think we have enough thumbs up here to merge, @xla

**Pull request merged**